### PR TITLE
Feat ts2js timeout

### PR DESCRIPTION
**Pull request by AlexGeller1**
Adds API to specify timeout when compiling Type Script

### CODE REVIEW

File: core/ts.core/build.properties [hunk 1]
```diff
@@ -1,5 +0,0 @@
-source.. = src/
```
Comment (lgrignon) [inline core/ts.core/build.properties]: why did this file and the following disappeared :/ 

Could you please keep only meaningful changes please?

File: core/ts.core/src/ts/client/ITypeScriptServiceClient.java [hunk 2]
```diff
@@ -123,6 +122,8 @@ void changeFile(String fileName, int line, int offset, int endLine, int endOffse
 
 	void updateFile(String fileName, String newText) throws TypeScriptException;
 
+	void updateFile(String fileName, String newText, int timeoutInSeconds) throws TypeScriptException;
```
Comment (lgrignon) [inline core/ts.core/src/ts/client/ITypeScriptServiceClient.java]: can we take unit please? Or at least milliseconds

File: core/ts.core/build.properties (hunk 1 above)
Comment (lgrignon) [reply core/ts.core/build.properties]: Oups, nevermind :)

Comment (lgrignon) [reply core/ts.core/build.properties]: I changed target branch and there are a lot less changes :) (It now points to mavenPackaging 11 branch)

**Comment by lgrignon** [comment]
You can now use version 2.0.4 with your modifications ;)
 http://repository.jsweet.org/artifactory/libs-release-local/org/jsweet/ext/typescript.java-ts.core/2.0.4/typescript.java-ts.core-2.0.4.jar

**Comment by AlexGeller1** [comment]
Thx!

**Pull request merged**